libs/common/src/sql-db-connection/migrations/1724342914301-AddFeeRecordReconciliationDataColumns.ts

```diff
@@ -0,0 +1,25 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddFeeRecordReconciliationDataColumns1724342914301 implements MigrationInterface {
```

Comment:
What is `1724342914301`?

Reply:
A timestamp - this is auto-generated by typeORM

Reply:
Sorry i'm not familiar with `typeORM` - why do we need timestamps in migration file names, functions and a `name` const?

Reply:
Because this is how typeorm generates migrations and keeps track of them in the database to avoid running the same migrations. I would recommend reading [the docs](https://orkhan.gitbook.io/typeorm/docs/migrations#creating-a-new-migration)